src/common/options/global.yaml.in

```diff
@@ -761,6 +761,12 @@ options:
   desc: Enable Intel QAT acceleration support for compression if available
   default: false
   with_legacy: true
+- name: qat_compressor_session_number
+  type: int
+  level: advanced
+  desc: Set the number of session within Qatzip when using QAT compressor
+  default: 200
+  with_legacy: true
```

Comment:
i'd avoid adding `with_legacy` conf vals. especially on the non-critical paths.

Reply:
I removed `with_legacy`.